### DISCUSSION TITLE
How do I get Visual Studio 2022 to recognize the installed package?

### DISCUSSION THREAD

**Discussion by Riprock** [microsoft/vcpkg#24130] [Q&A]
I am trying to use the Openssl library in my visual studio project and I followed the instructions for setting up vcpkg
```
All Done in an Administrator Powershell 
git clone https://github.com/microsoft/vcpkg
./vcpkg/bootstrap-vcpkg.sh\
 .\vcpkg\vcpkg.exe integrate install
.\vcpkg\vcpkg.exe install openssl-windows:x64-windows
```
![image](https://user-images.githubusercontent.com/8990377/163266622-f31486e3-3251-4e8e-9e4a-9d36f2b59d17.png)
I keep getting this error. I searched for the file in question and it exists in the vcpkg install directory in a subfolder
![image](https://user-images.githubusercontent.com/8990377/163266810-7ef891aa-0873-4dbb-9e7e-da7c81efb104.png)
Is there something im missing in order to use the library?

**Comment by tersion** [comment]
**Riprock**, I suppose you start your app from _Release_ or _Debug_ folder? In your case, you install dynamic link library type, not static. If so, you need put near .exe-file correspond .dll-file - **libcrypto-3.dll**

**Reply by Riprock** [reply]
I start my app from the builer and debugger in Visual studio. The screenshot above is just for note that I have the dll Installed
